doc/src/stdlib/collections.md

```diff
@@ -133,6 +133,9 @@ Base.collect(::Type, ::Any)
 Base.issubset(::Any, ::Any)
 Base.filter
 Base.filter!
+# this also includes the docstring for replace(A, old, new, n)
```

Comment:
I have no idea; if I include both signature, the docstring appear twice. This is very brittle, I must have done something wrong, would be nice if a documentation expert could help on this.